README.md

```diff
@@ -149,3 +149,10 @@ This exception contains a message string describing what went wrong, and also tw
 
 - `errors`: a list of errors that the server returned. In most cases, this array will contain only one element, an instance of ApiError, containing an error code and a message.
 - `requestId`: the identifier of the request you made. Please mention this to us when you have questions, as it will make debugging easier.
+
+## Rate limiting - exponential backoff
+
+This library supports [exponential backoff](https://en.wikipedia.org/wiki/Exponential_backoff).
+
+When you send too many concurrent requests, the server returns an error `429 - Too Many Requests`. We react to this by waiting for a while, and then retrying the request.
```

Comment:
I'd replace "we" with "the client", e.g. "the client reacts to this" instead of "we react to this"

Reply:
👍